src/bip44.c

```diff
@@ -158,7 +196,19 @@ bool bip44_isValidStakingKeyPath(const bip44_path_t* pathSpec)
 	return (bip44_getAddressValue(pathSpec) == 0);
 }
 
-// Futher
+#ifdef POOL_OPERATOR_APP
+bool bip44_isValidPoolColdKeyPath(const bip44_path_t* pathSpec)
+{
+#define CHECK(cond) if (!(cond)) return false
+	CHECK(pathSpec->length == BIP44_I_POOL_COLD_KEY + 1);
+	CHECK(bip44_hasValidCardanoPoolColdKeyPrefix(pathSpec));
+	CHECK(pathSpec->path[BIP44_I_POOL_COLD_KEY_USECASE] == 0);
```

Comment:
this should be hardened as per the officially approved version of CIP-1853: https://github.com/cardano-foundation/CIPs/blob/master/CIP-1853/CIP-1853.md